### ISSUE TITLE
Context deletes review comments from different context

### ISSUE THREAD

**Issue by davidtvs** [grantmcconnaughey/Lintly#30]
I've setup Lintly to run flake8 and black in my repository however after some tests I noticed that even though the two of them produce different statuses within a review I'll only see the review comments of the one that was last run.

So if I run flake8 first, I'll get flake8's review comments but then if I run black, all the comments from flake8 are deleted and I'll only see the ones from black.

I think a context should only delete comments that were created by that same context. Or, I would even prefer an option for nothing to be deleted, the comments are an important part of the review process.

**Comment by grantmcconnaughey** [comment]
Good catch! I hadn’t thought of this issue before. I think that should be an easy-ish fix. I use hidden HTML to identify which comments to delete. I’ll just need to add the context to that.

**Comment by davidtvs** [comment]
Nice 👍. What do you think of having an option that doesn't delete any of the previous comments? Personally, I would prefer that.

**Comment by grantmcconnaughey** [comment]
I think that makes a lot of sense, too. Now that GitHub hides previous comments as "resolved" it's less important to clean those up. In fact that would probably make more sense as the default behavior.
